.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
-  "extends": ["plugin:@typescript-eslint/recommended", "standard"],
+  "extends": [
+    "standard",
+    "plugin:@typescript-eslint/recommended"
```

Comment:
We could  perhaps add `overrides: { files: ["**/*.ts"] }` to only run the typescript rules for the relevent files? Not sure exactly what the difference would be.

Reply:
Oh, I think we actually want the typescript version of standard: https://github.com/standard/eslint-config-standard-with-typescript . Lemme see if that's a disruptive change.

Reply:
Haha yeah... Maybe as a separate migration.